### ISSUE TITLE
Improve error message

### ISSUE THREAD

**Issue by jasomdotnet** [allinurl/goaccess#492]
This is current version of error message:
```
GoAccess - version 1.0.2 - Jul  4 2016 19:36:15
Config file: /etc/goaccess.conf

Fatal error has occurred
Error occured at: src/goaccess.c - main - 1230
Nothing valid to process. Verify your date/time/log format.
```
Supply more informations like:

- Is time-format wrong?
- Is date-format wrong?
- Is log-format wrong? 
- Which part of log-format is wrong?
- Is anything missing from log-format?
- Which part of log-format cannot be parsed?

**Comment by allinurl** [comment]
Thanks for pointing this out. I have a branch that looks into this (still working on it). 

I'll close this since it will be addressed by issue #264.

**Comment by allinurl** [comment]
I added the ability to show which token didn't match log/date/time pattern or if any required fields are missing.

It will be pushed out in the upcoming release. Thanks again!
